src/algorithms/classical_segmentation/Dockerfile

```diff
@@ -12,4 +12,4 @@ RUN python -m pip install scipy==1.9.1 scikit-image==0.20.0
 # Precompile Python module
 RUN python -m compileall /bilayers/classical_segmentation.py
 
-CMD ["python"]
+CMD ["python"]
```

Comment:
Shouldn't have deleted the line break. Always leave a single newline at the end of files (code, text, config, etc). It's a POSIX requirement, among other reasons. See: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file